### ISSUE TITLE
Product card image missing alt attribute 

### ISSUE THREAD

**Issue by tomukasteris** [TryGhost/Source#67]
Hello,

Product card image missing alt attribute. I can't find where to add alt attribute.

**Comment by minimaluminium** [comment]
Similar to #68, this isn't a theme-specific issue. The product title should be used as alt text for the product card, will pass it to the team.
